bin/install_automation.sh

```diff
@@ -125,8 +125,8 @@ install_automation() {
 
     dbg "Configuring environment file $INSTALLATION_SOURCE/environment"
     cat <<EOF>"$INSTALLATION_SOURCE/environment"
-# Added on $(date --iso-8601=minutes) by $actual_inst_path/bin/$SCRIPT_FILENAME"
-# Any manual modifications will be lost upon upgrade or reinstall.
+# Added on $(date --utc --iso-8601=minutes) by $actual_inst_path/bin/$SCRIPT_FILENAME"
```

Comment:
LGTM. For future reference, `--iso-8601` already includes the UTC offset in `±HH:MM` form; adding `--utc` guarantees that this will be `+00:00`.

Reply:
Oh look at that, your right.  I saw "05:00" in a log and assumed the job ran at 5am. Durrr...it's [minus]05:00.

In any case, this will help prevent mental gymnastics since both the Cirrus-cron and github-actions are UTC based.

I'll update the commit message.